File: bro/src/main/java/me/xx2bab/bro/core/defaultor/DefaultMonitor.java
package me.xx2bab.bro.core.defaultor;


import me.xx2bab.bro.core.activity.Builder;
import me.xx2bab.bro.core.base.IBroMonitor;
import me.xx2bab.bro.core.util.BroRuntimeLog;

public class DefaultMonitor implements IBroMonitor {

    @Override
    public void onActivityRudderException(int errorCode, Builder builder) {

    }

    @Override
    public void onModuleException(int errorCode) {
        BroRuntimeLog.e("onModuleException: " + errorCode);
    }

    @Override
    public void onApiException(int errorCode) {
        BroRuntimeLog.e("onApiException: " + errorCode);
    }
}

File: bro/src/main/java/me/xx2bab/bro/core/defaultor/DefaultMonitor.kt
package me.xx2bab.bro.core.defaultor

import me.xx2bab.bro.core.activity.Builder
import me.xx2bab.bro.core.base.IBroMonitor
import me.xx2bab.bro.core.util.BroRuntimeLog.e

class DefaultMonitor : IBroMonitor {
    override fun onActivityRudderException(errorCode: Int, builder: Builder) {}
    override fun onModuleException(errorCode: Int) {
        e("onModuleException: $errorCode")
    }

    override fun onApiException(errorCode: Int) {
        e("onApiException: $errorCode")
    }
}